fenicsadapter/fenicsadapter.py

```diff
@@ -4,25 +4,20 @@
 :raise ImportError: if PRECICE_ROOT is not defined
 """
 import dolfin
-from dolfin import UserExpression, SubDomain, Function
+from dolfin import UserExpression, SubDomain, Function, Constant
 from scipy.interpolate import Rbf
 from scipy.interpolate import interp1d
 import numpy as np
 from .config import Config
+from .checkpointing import Checkpoint
+from .solverstate import SolverState
```

Comment:
The `.` indicates a *relative import* (see https://realpython.com/absolute-vs-relative-python-imports/). In this project it was the first thing that came to my mind. As far as I understand an *absolute import* should also work (instead of  `from .config import Config` one can use the statement `from fenicsadapter.config import Config`). 

As far as I understand the article above, the main reasons for using relative imports are a better readability and more flexibility when it comes to moving packages/subpackages within the project. This means, we could indeed switch from relative to absolute imports, but I think in the end it does not really matter.